doc/Programming-guide/Key-platform-features/Endpoint-profiles/index.md

```diff
@@ -373,42 +382,39 @@ For the purpose of this guide we will use a fairly abstract server-side profile
+To export a client-side EP profile schema, click the corresponding **Export** button and choose the export method from the drop-down list.
+See [CT schema export support]({{root_url}}Programming-guide/Key-platform-features/Common-Type-Library/#ct-schema-export-support) for available export methods details.
+
+As a tenant developer, you can create new server-side EP schemas for your application as follows:
+
+1. Under the **Schemas** section of the application, click **Server-side EP profile schemas**, then click the **Add schema** button.
+
+2. On the **Add profile schema** page, enter your schema name and description (optional).
 
-1. In the **Server-side EP profile schemas** window for the application, click **Add schema**.
-2. In the **Add profile schema** window enter the name of the schema.
 3. Then create a schema using one of the two options:
 
-    1. Using the existing CT by clicking **Select existing type** and selecting exiting CT version from FQN and version drop-downs.
+    * Use the existing [common type (CT)]({{root_url}}Glossary/#common-type-ct) by clicking **Select existing type**.
+    Click the **Select fully qualified name of existing type** field and select the CT from the drop-down list, then select the version from the corresponding drop-down list.
     
     <img src="admin-ui/Create server-side endpoint profile schema 1.png">
     
-    2. Create new CT by clicking **Create new type**. In this case you will be redirected to **Add new type** window. Here you can create a schema either by using the 
-    schema form or by uploading a schema in the [Avro](http://avro.apache.org/docs/current/spec.html) format from a file.
+    * Create new CT by clicking **Create new type**.
+    The **Add new type** page will open.
+    Here you can create a schema either by using the schema form or by uploading a file containing the schema in the [Avro](http://avro.apache.org/docs/current/spec.html) format.
     
     <img src="admin-ui/Create server-side endpoint profile schema 2.png">
     
-4. Click **Add** at the top of the window to save the schema.
-
-If you want to review the added Avro schema, open the corresponding **Server-side EP profile schema** window by clicking the schema in the **Server-side EP profile schemas** window.
+4. Click **Add** to save your schema.
 
-<img src="admin-ui/View server-side endpoint profile schema.png">
+### REST API for server-side EP profile
 
-### REST API for Server-side endpoint profile
-
-Visit [Admin REST API]({{root_url}}Programming-guide/Server-REST-APIs/#/Profiling) documentation page for detailed description of the REST API, 
-its purpose, interfaces and features supported.
-
-Admin REST API provides the following actions:
+Use the server REST API to perform the following actions:
 
 * [Get profile based on endpoint key]({{root_url}}Programming-guide/Server-REST-APIs/#!/Profiling/getEndpointProfileByKeyHash)
```

Comment:
This very same API call was listed as client-side EP profile related in line 139. Is this intentional?